Stratis.Bitcoin/FullNode.cs

```diff
@@ -133,6 +143,13 @@ void RunLoop()
 						//TODO: 
 						Logs.FullNode.LogError("Block rejected: " + block.Error.Message);
 					}
+
+					if (!reorg && block.Error == null)
```

Comment:
> hmm but there is no await in the consensus loop so the task is just fired and the loop continues.

Exactly, this is best as it does not slow the consensus loop, and protect against unwanted exception. What do you want more ?